package.xml

```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0"?>
 <package>
-  <name>snap_ros_examples</name>
+  <name>snapdragon_mavros_vislam</name>
   <version>0.1.1</version>
-  <description>This package provides example code as ros nodes for select features of snapdragon flight platfrom(TM)</description>
+  <description>
+    Fork of the Qualcomm Vislam example code from ATLFlight (https://github.com/ATLFlight/ros-examples) modified for use with the px4 flight stack.
+  </description>
 
   <!-- One maintainer tag required, multiple allowed, one person per tag --> 
   <!-- Example:  -->
   <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
-  <maintainer email="rkintada@gmail.com">rkintada</maintainer>
+  <maintainer email="potaito@TODO.com">potaito</maintainer>
```

Comment:
I think this can be undone and reverted to `rkitanda` :smile: 
I had bigger plans for this back in the days ....